babynames.py

```diff
@@ -66,15 +87,18 @@ def main(args):
     parser = create_parser()
     # Run the parser to collect command-line arguments into a NAMESPACE called 'ns'
     ns = parser.parse_args(args)
-
     if not ns:
         parser.print_usage()
         sys.exit(1)
 
     file_list = ns.files
-
+    for file_ in file_list:
+        text = extract_names(file_)
+        
     # option flag
     create_summary = ns.summaryfile
+    with open("baby{}.html.summary".format(text[0:5]), "w") as f:
```

Comment:
Make sure you are only creating .summary files if the "--summaryfile" flag is present on the command line.